taskcluster/ci/generate-screenshots/kind.yml

```diff
@@ -5,46 +5,22 @@
 loader: taskgraph.loader.transform:loader
 
 transforms:
-    - ffios_taskgraph.transforms.screenshots:transforms
+    - ffios_taskgraph.transforms.bitrise:transforms
     - ffios_taskgraph.transforms.secrets:transforms
     - taskgraph.transforms.job:transforms
     - taskgraph.transforms.task:transforms
 
+kind-dependencies:
+    - build
 
 job-defaults:
+    bitrise-workflow: jlorenzo_L10nScreenshotsTests
```

Comment:
I updated this workflow to not compile anymore. Instead it fetches `build-derived-data-path` and unzip it at the right place.